pymodbus/repl/client/mclient.py

```diff
@@ -321,13 +321,14 @@ def report_slave_id(self, slave=Defaults.Slave, **kwargs):
             }
         return ExtendedRequestSupport._process_exception(resp)
 
-    def read_exception_status(self, **kwargs):
```

Comment:
I think this was previously broken

Reply:
Could very well be, but then lets fix it.

Reply:
This commit does exactly that.